lib/ruby_lsp/ruby_lsp_rails/server.rb

```diff
@@ -119,12 +119,17 @@ def execute(request, params)
           require params[:server_addon_path]
           ServerAddon.finalize_registrations!(@stdout)
         when "server_addon/delegate"
-          server_addon_name = params.delete(:server_addon_name)
-          request_name = params.delete(:request_name)
-          ServerAddon.delegate(server_addon_name, request_name, params)
+          server_addon_name = params[:server_addon_name]
+          request_name = params[:request_name]
+          ServerAddon.delegate(server_addon_name, request_name, params.except(:request_name, :server_addon_name))
         end
+        request_name = request
+        request_name = "#{params[:server_addon_name]}##{params[:request_name]}" if request == "server_addon/delegate"
+      # Since this is a common problem, we show a specific error message to the user, instead of the full stack trace.
+      rescue ActiveRecord::ConnectionNotEstablished
+        log_message("Request #{request_name} failed because database connection was not established.")
       rescue => e
-        send_message({ error: e.full_message(highlight: false) })
+        log_message("Request #{request_name} failed:\n" + e.full_message(highlight: false))
```

Comment:
@vinistock I updates this so that we show the request name for both `rescue` clauses.